docs/api/1.x/resource.rst

```diff
@@ -427,18 +200,6 @@ A ``details`` attribute in the response provides the offending record and
 field name. See :ref:`dedicated section about errors <error-responses>`.
 
 
-Timestamp
----------
-
-When a record is created, the timestamp of the collection is incremented.
-
-It is possible to force the timestamp if the specified record has a
-``last_modified`` field.
-
-If the specified timestamp is in the past, the collection timestamp does not
-take the value of the created record but is bumped into the future as usual.
-
-
```

Comment:
You're a better man than I if you can follow a giant documentation merge just by looking at one diff :)

The contents of `resource.rst` are intentionally general and try to set out principles that all endpoints follow. I think it might be better to just remove the entire file and describe the actual endpoints in buckets, collections and records files. In that case, my goal is to eventually delete this entire file, so I don't want to put a link here. But I'm not sure, and I would be open to the argument that having a generic file like this is valuable. Let's iterate on it.